repository/t.json

```diff
@@ -3185,6 +3185,20 @@
 				}
 			]
 		},
+		{
+			"name": "Translator (multi-engine, fast)",
+			"details": "https://github.com/dmytrovoytko/sublimetext-translate",
+			"author": "Dmytro Voytko",
+			"homepage": "https://github.com/dmytrovoytko/sublimetext-translate",
+			"readme": "https://github.com/dmytrovoytko/SublimeText-Translate/blob/master/README.md",
```

Comment:
You don’t need to specify homepage and readme if they’re just the defaults for GitHub.

Reply:
So please remove them.

Reply:
Done!